ddtrace/tracer/option.go

```diff
@@ -480,6 +486,30 @@ func WithLogStartup(enabled bool) StartOption {
 	}
 }
 
+// WithProfilerCodeHotspots enables the code hotspots integration between
+// tracing and profiling. This is done by automatically attaching goroutine
+// labels called "span id" and "local root span id" when new spans are created.
+// You should not use these label names in your own code.
+// TODO(fg) finalize name, decide if one option for both features is enough,
```

Comment:
Maybe this shouldn't show up on `godoc`? WDYT? Maybe moving it inside the function will help with that.